asreview/webapp/start_flask.py

```diff
@@ -299,6 +299,11 @@ def create_app(**kwargs):
     login_manager.session_protection = "strong"
 
     if app.config["AUTHENTICATION_ENABLED"] is False:
+        # This ensures the app handles the anonymous user
+        # when authentication is disabled and there is no
+        # configuration file
+        app.config["SECRET_KEY"] = "no_secret"
```

Comment:
```suggestion
        app.config["SECRET_KEY"] = None
```

Does this also work?

Reply:
None doesn't work, empty string does. And I think the latter looks better than "no_secret".